src/core_plugins/kibana/public/discover/controllers/discover.js

```diff
@@ -465,8 +461,8 @@ function discoverController(
     if ($state.query.language && $state.query.language !== query.language) {
```

Comment:
I called these methods `updateQuery` since they only update state. Now that they also fetch results, maybe we should call them something like `fetchWithQuery`?

Reply:
:+1: 9c27c3340

Reply:
That name sounds good to me! Did you forget to update all the usages though?

Reply:
How embarrassing. ef09fa05c